apps/fluent-tester/package.json

```
@@ -29,6 +29,7 @@
     "@fluentui-react-native/pressable": ">=0.3.76 <1.0.0",
     "@fluentui-react-native/focus-trap-zone": ">=0.4.0 <1.0.0",
     "@fluentui-react-native/link": ">=0.8.0 <1.0.0",
+    "@fluentui-react-native/shimmer": ">=0.1.0 <1.0.0",
     "@fluentui-react-native/radio-group": ">=0.3.76 <1.0.0",
     "@fluentui-react-native/persona-coin": ">=0.3.76 <1.0.0",
     "@fluentui-react-native/persona": ">=0.5.73 <1.0.0",
```

Does this actually pull code from npm? I was under the impression that we should be using `*` so that `npm`/`yarn` would get the packages from the monorepo instead.
```suggestion
    "@fluentui-react-native/pressable": "*",
    "@fluentui-react-native/focus-trap-zone": "*",
    "@fluentui-react-native/link": "*",
    "@fluentui-react-native/shimmer": "*",
    "@fluentui-react-native/radio-group": "*",
    "@fluentui-react-native/persona-coin": "*",
    "@fluentui-react-native/persona": "*",
```

---

It's more about the externally facing behavior, internally it just uses local versions. The >=0.1.0 <1.0.0 syntax emulates the behavior of ^1.x.y for packages that haven't been major released yet. The specific ranges aid the bundler in making the right decision for customers, these numbers get updated by beachball during publish. As for switching to * I'm less clear on what that would do, but I'm hesitant to make the change without more thinking on what will happen.